bat-utils/lib/hapi-server.js

```diff
@@ -76,6 +76,12 @@ const Server = async (options, runtime) => {
     })
   }
 
+  server.route({
```

Comment:
unrelated?

Reply:
yes, must have forgotten to take out. removed